torchvision/datasets/cub2011.py

```diff
@@ -0,0 +1,31 @@
+import os
+import torch
+import torch.utils.data as data
+import torchvision
+from torchvision.datasets import ImageFolder
+from torchvision.datasets import CIFAR10
+
+class Cub2011(ImageFolder, CIFAR10):
```

Comment:
should be all uppercase I think `CUB2011`

Reply:
Not sure you can leverage the `CIFAR10` dataset here as it assumes 60k images internally with a 50k/10k split. However, CUB2011 only has 11,788 images.